wrapper/src/main/java/software/amazon/jdbc/plugin/staledns/RdsStaleDnsHelper.java

```diff
@@ -76,7 +76,7 @@ public Connection getVerifiedConnection(
     }
 
     final String hostInetAddress = clusterInetAddress;
-    LOGGER.finest(() -> Messages.get("AuroraStaleDnsHelper.clusterEndpointDns",
+    LOGGER.finest(() -> Messages.get("RdsStaleDnsHelper.clusterEndpointDns",
         new Object[]{hostInetAddress}));
```

Comment:
Corresponding changes should be done in `messages.properties` file.  This comment applies to all places below where a message key changes.